dhcp/dhcp.go

```diff
@@ -11,7 +11,7 @@ type Reply interface {
 }
 
 func NewReply(w dhcp4.ReplyWriter, req *dhcp4.Packet) Reply {
-	switch req.GetMessageType() {
+	switch req.GetMessageType() { //nolint:exhaustive // Boots only supports responding to these message types
```

Comment:
you could also do a `default:` case that doesn't do anything. This way one less `//nolint`.

Reply:
makes sense. updated.